tests/init_test.go

```diff
@@ -140,6 +140,11 @@ func (tm *testMatcher) whitelist(pattern string) {
 	tm.whitelistpat = regexp.MustCompile(pattern)
 }
 
+func TestDummyMatcherToKeepBothStaticcheckandHolimanHappy(t *testing.T) {
```

Comment:
This test name doesn't work for me.

Reply:
and what test name would work for you?

Reply:
`TestMatcherWhitelist` maybe?

Reply:
github didn't refresh, didn't see your suggestion :angry: